### DISCUSSION TITLE
Possible error: Chapter 2, Problem 69c

### DISCUSSION THREAD

**Discussion by bryzhang** [fifthist/Introduction-To-Probability-Blitzstein-Solutions#27] [General]
Hi, I'm pretty sure the arithmetic in the aforementioned problem should resolve to d = (4y + p - 1) / 4p. Please take a look and let me know.

**Comment by JoeZiminski** [comment]
I agree, I attempted this question and got:


![image](https://user-images.githubusercontent.com/55797454/212875804-93f52365-4c8c-4730-8dc2-332467b2de03.png)
